core/web/log_controller.go

```diff
@@ -0,0 +1,101 @@
+package web
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/gin-gonic/gin"
+	"github.com/smartcontractkit/chainlink/core/logger"
+	"github.com/smartcontractkit/chainlink/core/services/chainlink"
+	"github.com/smartcontractkit/chainlink/core/web/presenters"
+	"go.uber.org/zap/zapcore"
+)
+
+// LogController manages the logger config
+type LogController struct {
+	App chainlink.Application
+}
+
+type LoglevelPatchRequest struct {
+	LogLevel string `json:"logLevel"`
+	LogSql   string `json:"logSql"`
```

Comment:
This should be consistent with the response and be a boolean and update the naming.

You may need a pointer to the boolean since it can be omitted

```suggestion
	SqlEnabled   string `json:"sqlEnabled"`
```